### ISSUE TITLE
use thumbnails.odysee.com as thumb cdn

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-android#1229]
Also make sure we're using the resizing method- maybe use a size similar to odysee.com mobile (please check code)

**Comment by kekkyojin** [comment]
This was fixed on Odysee Android but it wasn’t ported to LBRY Android

**Comment by tzarebczan** [comment]
I think that one may be using the image-optimizer URL? Can you also confirm we are passing a width/height/quality params? Would be good to sync those up with mobile odysee.com so one set can be cached properly.

**Comment by kekkyojin** [comment]
Yes, @tzarebczan . On Odysee Android, thumbnail.odysee.com/optimize url is being used to get an image file. Width and height are passed to that url, along quality value, which currently is set to 85 over there.

https://github.com/OdyseeTeam/odysee-android/blob/042fc4daff8a60a4430e06c7b83ad6fff7b2bf56/app/src/main/java/com/odysee/app/model/Claim.java#L555
